pkg/apis/kops/v1alpha2/networking.go

```diff
@@ -38,21 +38,21 @@ type NetworkingSpec struct {
 }
 
 // ClassicNetworkingSpec is the specification of classic networking mode, integrated into kubernetes
```

Comment:
```suggestion
// ClassicNetworkingSpec is the specification of classic networking mode, integrated into kubernetes.
```

Reply:
should these changes also be made to `pkg/apis/kops/networking.go`?

Reply:
They should :)